config/database.yml

```diff
@@ -0,0 +1,31 @@
+development:
```

Comment:
You can update dev and test to use postgersql if you want.
Or just leave it at sqlite3.